packages/paste-website/src/styles/docSearch.ts

```diff
@@ -0,0 +1,165 @@
+import {css} from '@twilio-paste/styling-library';
+
+export const docSearchStyles = css({
+  '.algolia-autocomplete': {
+    width: '100%',
+  },
+  '.algolia-autocomplete .ds-dropdown-menu .ds-suggestion': {
+    marginY: 'space30',
+    marginX: '0',
+  },
+  '.algolia-autocomplete .ds-dropdown-menu .ds-suggestions a': {
+    textDecoration: 'none',
+  },
+  '.algolia-autocomplete .ds-dropdown-menu .ds-suggestions a:hover, .algolia-autocomplete .ds-dropdown-menu .ds-suggestions a:focus': {
+    textDecoration: 'none',
+  },
+  '.algolia-autocomplete .ds-dropdown-menu .ds-suggestions a:hover .algolia-docsearch-suggestion--title, .algolia-autocomplete .ds-dropdown-menu .ds-suggestions a:focus .algolia-docsearch-suggestion--title': {
+    textDecoration: 'underline',
+  },
+  '.algolia-autocomplete .algolia-docsearch-suggestion.algolia-docsearch-suggestion__main .algolia-docsearch-suggestion--category-header': {
+    display: 'none',
+  },
+  '.algolia-autocomplete .algolia-docsearch-suggestion--category-header': {
+    display: 'none',
+  },
+  '.algolia-autocomplete .algolia-docsearch-suggestion--category-header .algolia-docsearch-suggestion--category-header-lvl0 .algolia-docsearch-suggestion--highlight': {
+    // boxShadow: 'inset 0 -2px 0 0 colorBorderPrimary',
```

Comment:
Should this be dropped?

Reply:
Oooh good spot, it didn't seem to do anything so yes